CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `--service.registry.mirrors` flag for setting registry mirror domains.
+- Replace `--service.registrydomain` with `--service.registry.domain`.
```

Comment:
Doesn't this mean something changed? It is in the added section.

Reply:
I agree but the way we do majors with K8s minors gives me a headache here. Should it be `### Changed` even though we'll release next minor?

Reply:
We can fix it after merge. Or even release.

Reply:
I know where you are coming from and this screws your automation ideas to some extend. Having changes in the changelog "added" section does just not make any sense semantically.

Reply:
So yes, please change that.

Reply:
https://github.com/giantswarm/aws-operator/pull/2585